homework.md

```diff
@@ -4,77 +4,77 @@
 
 What is 2 + 2?
 
-5
+4
 
 ## Question 2
 
 What is the opposite of Black?
 
-Purple
+White
 
 ## Question 3
 
 What is the  2 * 3 * 6 * 1 * 10 * 34?
 
-1322
+12,240
 
 ## Question 4 
 
 What does HTML stand for?
 
-Hyper Tall Massive Language
+Hyper Text Markup Language 
 
 ## Question 5
 
 What is CSS useful for?
 
-Making a website go faster
+It is the styling of the website i.e. the colours, the fonts etc. 
 
 ## Question 6
 
 Using CSS, how would I select every `<h1>` tag on the page?
 
-```css
-#h1 {
-
+h1 {
+    whatever command you would like. 
 }
-```
 
 ## Question 7
 
 What is JavaScript?
 
-A programming language used on the internet
+It is a text-based programming language. 
 
 ## Question 8
 
 What three problem does Git & Github solve?
 
-When people want to show off code to each other they can put it on Github
+1. It allows for easier contributions and to manage multiple edits simultaneously.
+2. It allows contributors to make changes or work on their own forked repositories. 
+3. It enables each user to have control whilst working colaboratively. 
 
 ## Question 9
 
 What happens when you `fork` a repository?
 
-You delete it
+When you copy the original repo. 
 
 ## Question 10 
 
 What happens when you clone a repostory?
 
-It send it to a friend
+Is where you copy the repo to your local machine. 
 
 ## Question 11
 
 What is a Pull Request?
```

Comment:
Hey Monique, nice work. I see you left the personal details and home work details empty at the start. You might have done it intentionally or haven't notice, adding that would be great as they ask in the assignment.

Reply:
Hi Amanuel, I did miss those. Thank you for noticing, I have corrected :)